docs/src/Experimental/Singularities/space_germs.md

```diff
@@ -0,0 +1,208 @@
+   germ_at-point(X::AffineScheme, I::Ideal)
+   ```
+   where I is a (maximal) ideal describing the chosen kk-point on the affine kk-scheme.  
+
+   **Provides:** SpaceGerm, restriction map.
+
+
+ * localization of a polynomial ring at a point
+
+   ```julia
+   germ_at_point(R::MPolyRing, I::MPolyIdeal)
+   germ_at_point(R::MPolyQuo, I::MPolyIdeal)
+   ```
+   where I is a maximal ideal describing the chosen base_ring(R)-point.
+
+   **Provides:** space germ and restriction map.
+
+ * localized ring with respect to complement of prime ideal or complement of maximal ideal)
+
+   ```julia
+   SpaceGerm(R::MPolyLocalizedRing)
+   SpaceGerm(R::MPolyQuoLocalizedRing)
+   ```
+   **Provides:** Space germ
+   ```julia
+   germ_at_point(R::MPolyLocalizedRing)
+   germ_at_point(R::MPolyQuoLocalizedRing)
+   ```
+
+   **Provides:** Space germ, restriction map  
+
+   (point inherited from underlying local ring)
+
+Morphisms of space germs are morphisms of the underlying local rings and hence
+also handled in this way in Oscar.
+
+## Basic functionality for space germs
+
+Most of the basic functionality immediately falls back to the underlying 
+affine algebra or affine scheme and is just provided on the geometric side 
+for convenience and consistence of functionality:
+
+### internal data of a space germ  
+
+ * Pass from the germ $(X,x)$ back to some affine scheme $X$ with the appropriate localization at $x$ by
+   ```julia
+   representative(X::SpaceGerm)
+   ```
+   **Provides:** AffineScheme   
+
+ * Given the space germ $(X,x)$, the point $x$ is returned by:
+
+   ```julia
+   point(X:SpaceGerm)
+   ```
+   **Provides:** Ideal
+
+!!! note
+    The returned ideal is a prime ideal in the ring of a representative of the germ. At first use of it or at the latest upon the first call of representative, the respective affine scheme is cached and subsequently used for all further purposes requiring a representative
+
+ 
+ * Given a space germ $(X,x)$, the corresponding local ring ${\mathcal O}_{(X,x)}$ is returned by:
+   ```julia
+   ring(X::SpaceGerm)
+   ```
+   **Provides:** MPolyQuoLocalizedRing
+
+ * Analogously, the modulus of the ring of a given germ $(X,x)$ can be obtained by:
+   ```julia
+   ideal(X::SpaceGerm)
+   ```
+   **Provides:** Ideal
+
+ * For technical reasons all germs $(X,x)$ in OSCAR are embedded and the smooth ambient germ can be accessed by:
+   ```julia
+   ambient_germ(X::SpaceGerm)
+   ```
+   **Provides:** SpaceGerm
+
+### containment/equality of space germs
+    
+ * containment  
+
+   ```julia
+   issubset(X::SpaceGerm, Y::SpaceGerm)
+   ```
+
+   Test whether X is a subgerm of Y.  
+
+   **Provides:** Boolean Value
+ 
+!!! note
+    The name 'issubset' has been chosen for consistency with other types, but is slightly misleading, as the function does not only test containment of the underlying sets, but in the scheme-theoretic sense.
+
+ * equality
+
+   ```julia
+   is_canonically_isomorphic(X::SpaceGerm, Y::SpaceGerm)
```

Comment:
Has been removed and replaced by `==` by @simonbrandhorst .

Reply:
corrected.